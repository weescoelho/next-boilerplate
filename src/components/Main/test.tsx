import { render, screen } from '@testing-library/react';
import { Main } from '.';

describe('<Main/>', () => {
  it('should render main component', () => {
    const { container } = render(<Main />);
    expect(
      screen.getByRole('heading', { name: /Next Boilerplate/i })
    ).toBeInTheDocument();
    expect(container.firstChild).toMatchSnapshot();
  });
  it('should render the colors correctly', () => {
    const { container } = render(<Main />);
    expect(container.firstChild).toHaveStyle({ color: 'blueviolet' });
  });
});
