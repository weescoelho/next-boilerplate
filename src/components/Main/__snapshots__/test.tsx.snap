// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main/> should render main component 1`] = `
.c0 {
  color: blueviolet;
  font-size: 5rem;
}

<h1
  class="c0"
>
  Next Boilerplate
</h1>
`;
